src/components/organisms/ConceptPicker.tsx

```diff
@@ -0,0 +1,167 @@
+import { useEffect, useRef, useState } from "react";
+import { NextRouter, useRouter } from "next/router";
+
+import { Label } from "@/components/labels/Label";
+import { InputCheck } from "@/components/forms/Checkbox";
+import { Select } from "@/components/atoms/select/Select";
+
+import { fetchAndProcessConcepts } from "@/utils/processConcepts";
+import { groupByRootConcept } from "@/utils/conceptsGroupedbyRootConcept";
+
+import { QUERY_PARAMS } from "@/constants/queryParams";
+
+import { TConcept } from "@/types";
+
+type TProps = {
+  concepts: TConcept[];
+  startingSort?: TSort;
+};
+
+type TSort = "A-Z" | "Grouped";
+
+const SORT_OPTIONS: TSort[] = ["A-Z", "Grouped"];
```

Comment:
Suggestion: Do this the other way round - derive `TSort` from `SORT_OPTIONS` ([example](https://stackoverflow.com/a/55505556)).

Reply:
Yes! Great catch, meant to come back to this 🙏